sdk/communication/azure-communication-callingserver/azure/communication/callingserver/aio/_callingserver_client_async.py

```diff
@@ -41,6 +41,8 @@
     from azure.core.credentials_async import AsyncTokenCredential
```

Comment:
this _async file within aio folder is for async client. Please create async download and recording apis.
and for the test, please use file name with _async as postfix. The format of async files are run in python3 only and won't run for Python 2 backward compatible. You can take other _async files as references. 
Suggestion:
You can copy the sync api and test you wrote already, and modified them to async format.

Reply:
Async download implementation is taken care by Franko @cochi2

Reply:
I already implemented async operations. Waiting for this PR to close to merge and push a new PR.